website/content/docs/integrations/consul/service-mesh.mdx

```diff
@@ -159,6 +159,7 @@ Using transparent proxy has several important requirements:
   `client_addr`).
 * The Consul agent must be configured with [`recursors`][] if you want
   allocations to make DNS queries for applications outside the service mesh.
+* Your workload's task cannot use the same UID as the Envoy sidecar proxy.
```

Comment:
worth linking to the `uid` parameter on the jobspec page, to connect the dots across documents?  it also specifies the default "101" there, which might be a nice detail to know for someone who hits this situation.